bigquery/api/GettingStarted/BigqueryTest/BigqueryTest.cs

```diff
@@ -0,0 +1,623 @@
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+// [START all]
+
+using System;
+// [START create_bigquery_client]
+using Google.Bigquery.V2;
+// [END create_bigquery_client]
+using Google.Storage.V1;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using Xunit;
+using System.IO;
+using System.Diagnostics;
+
+namespace GoogleCloudSamples
+{
+    public class BigqueryTest
+    {
+        private readonly string _projectId;
+        private readonly BigqueryClient _client;
+
+        public BigqueryTest()
+        {
+            // [START setup]
+            // By default, the Google.Bigquery.V2 library client will authenticate 
+            // using the service account file (created in the Google Developers 
+            // Console) specified by the GOOGLE_APPLICATION_CREDENTIALS 
+            // environment variable. If you are running on
+            // a Google Compute Engine VM, authentication is completely 
+            // automatic.
+            _projectId = Environment.GetEnvironmentVariable("GOOGLE_PROJECT_ID");
+            _client = BigqueryClient.Create(_projectId);
+            // [END setup]
+        }
+
+
+        public void CreateDataset(string datasetId, BigqueryClient client)
+        {
+            var dataset = client.GetOrCreateDataset(datasetId);
+        }
+
+        // [START create_table]
+        public void CreateTable(string datasetId, string tableId, BigqueryClient client)
+        {
+            var dataset = client.GetDataset(datasetId);
+            // Create schema for new table.
+            var schema = new TableSchemaBuilder
+            {
+                { "title", BigqueryDbType.String },
+                { "unique_words", BigqueryDbType.Integer }
+            }.Build();
+            // Create the table if it doesn't exist.
+            BigqueryTable table = dataset.GetOrCreateTable(tableId, schema);
+        }
+        // [END create_table]
+
+        public BigqueryDataset GetDataset(string datasetId, BigqueryClient client)
+        {
+            var dataset = client.GetDataset(datasetId);
+            return dataset;
+        }
+        public BigqueryTable GetTable(string datasetId, string tableId, BigqueryClient client)
+        {
+            var table = client.GetTable(datasetId, tableId);
+            return table;
+        }
+
+        public BigqueryTable GetTable(string projectId, string datasetId, string tableId, BigqueryClient client)
+        {
+            var table = client.GetTable(projectId, datasetId, tableId);
+            return table;
+        }
+
+        // [START list_tables]
+        public List<BigqueryTable> ListTables(string datasetId, BigqueryClient client)

```

Comment:
What do you think of the briefer syntax?
```
        public List<BigqueryTable> ListTables(string datasetId, BigqueryClient client) =>
           client.ListTables(datasetId).ToList();
```

But I'm also wondering if this function should be inlined.